server/channels/app/post.go

```diff
@@ -1466,7 +1466,7 @@ func (a *App) searchPostsInTeam(teamID string, userID string, paramsList []*mode
 		if result.NErr != nil {
 			return nil, model.NewAppError("searchPostsInTeam", "app.post.search.app_error", nil, "", http.StatusInternalServerError).Wrap(result.NErr)
 		}
-		data := result.Data.(*model.PostList)
+		data := result.Data
 		posts.Extend(data)
```

Comment:
Nit pick: It might more clear to use a more specific variable name here
```suggestion
		posts := result.Data
		posts.Extend(posts)
```

Reply:
Or even:
```suggestion
		posts.Extend(result.Data)
```